pkg/api/plugins.go

```diff
@@ -562,3 +562,33 @@ func mdFilepath(mdFilename string) (string, error) {
 		return "", ErrUnexpectedFileExtension
 	}
 }
+
+func (hs *HTTPServer) CheckEnabled(c *contextmodel.ReqContext) {
```

Comment:
Nit: Just because the `HTTPServer` struct is a mammoth by now, I'd change this to take in `pluginStore` and `pluginSettings` structs/interfaces.

Reply:
:+1: moved to its own file too